src/helpers/enrich-dataset.test.ts

```diff
@@ -0,0 +1,585 @@
+
+        const geojson = {
+            type: 'Feature',
+            properties: hubDataset
+        }
+
+        const { properties } = enrichDataset(geojson, hubsite);
+        expect(properties.accessUrlWMS).toBe('https://services1.arcgis.com/jUJYIo9tSA7EHvfZ/arcgis/services/DCAT_Test/FeatureServer/WMSServer?request=GetCapabilities&service=WMS');
+    });
+    
+
+    it('should generate download link without query string if wkid is not present in spatialReference', () => {
+        const hubDataset = {
+            id: 'foo',
+            access: 'public',
+            slug: 'nissan::skyline-gtr',
+            size: 1,
+            type: 'CSV',
+            created: 1570747289000,
+            license: 'none',
+            url: 'https://services1.arcgis.com/jUJYIo9tSA7EHvfZ/arcgis/rest/services/DCAT_Test/FeatureServer/0',
+            layer: {
+                geometryType: 'esriGeometryPolygon',
+            },
+            server: {
+                spatialReference: {
+                    latestWkid: 3310,
+                },
+            },
+            identifier: 'CALFIRE::DCAT_Test',
+            supportedExtensions: ['WMSServer'],
+            boundary: {
+                geometry: {
+                    type: 'sd'
+                }
+            }
+        };
+
+        const geojson = {
+            type: 'Feature',
+            properties: hubDataset
+        }
+
+        const { properties } = enrichDataset(geojson, hubsite);
+        expect(properties.accessUrlWMS).toBe('https://services1.arcgis.com/jUJYIo9tSA7EHvfZ/arcgis/services/DCAT_Test/FeatureServer/WMSServer?request=GetCapabilities&service=WMS');
+    });
+
+    it('should return structuredLicense url if dataset contains uninterpolated string', () => {
+        const hubDataset = {
+            id: 'foo',
+            access: 'public',
+            slug: 'nissan::skyline-gtr',
+            size: 1,
+            type: 'CSV',
+            created: 1570747289000,
+            license: '{{customLicense}}',
+            structuredLicense: { url: 'arcgis.com' }
+        };
+
+        const geojson = {
+            type: 'Feature',
+            properties: hubDataset
+        }
+
+        const { properties } = enrichDataset(geojson, hubsite);
+        expect(properties.license).toBe('arcgis.com');
+    });
+
+    it('should return custom license if avaiable', () => {
+        const hubDataset = {
+            id: 'foo',
+            access: 'public',
+            slug: 'nissan::skyline-gtr',
+            size: 1,
+            type: 'CSV',
+            created: 1570747289000,
+            license: 'customLicense',
+            url: 'arcgis.com'
+        };
+
+        const geojson = {
+            type: 'Feature',
+            properties: hubDataset
+        }
+
+        const { properties } = enrichDataset(geojson, hubsite);
+        expect(properties.license).toBe('customLicense');
+    });
+
+    it('should get csv access url if dataset is a proxied csv', () => {
```

Comment:
This name is misleading, it doesn't even test the csvAccessUrl

Reply:
Removed the test as code for proxied CSV URL has been deleted.